crates/cnidarium/src/rpc.rs

```diff
@@ -121,17 +122,29 @@ impl QueryService for Server {
 
         const MAX_REGEX_LEN: usize = 1024;
```

Comment:
> I am not sure if our current size limit (1024) is reasonable:

the description in #4360 questioned this limit, should we still bump this to something like..

```suggestion
        const MAX_REGEX_LEN: usize = 4096;
```

Reply:
In practice I think it might be okay as-is. The common regexes you'll want to use are simple and short prefix-style: `^dex/ra/` `^staking/` etc. and I only encountered this because I was "holding it wrong" and supplied a wildcard character match unnecessarily.

If we bump against it in the future we can increase it.

Reply:
sounds good! marking as resolved now that we have closed this loop ➰